app/views/subscription/index.phtml

```diff
@@ -59,29 +59,33 @@
 							$empty_title = _t('sub.feed.empty');
 						}
 						$mute_class = $feed->mute() ? ' mute' : '';
+
+						$title = $error_title !== '' ? '⚠ ' . $error_title . '&#13;' : '';
```

Comment:
```suggestion
						$title = $error_title !== '' ? '⚠ ' . $error_title . "\n" : '';
```

Doesn't that work? (not tested) I would prefer, rather than en entity

Reply:
it does not work (Firefox on Windows):
![grafik](https://github.com/user-attachments/assets/0eb65397-4a0d-42f0-8701-244b8dd6a7b2)

Reply:
Please make sure to try with `"\n"` (double quotes) or `PHP_EOL` and not single quotes.
Seems to work for me.

Reply:
ok. I got it. Thanks for clarification.

Yes it works, because it adds a real line break into the `title` attribute. It means that the HTML tag will have a line break inside of its tag.
IMHO I am not a big fan of having breaks in HTML attributes. And it messes a bit the PHP code with single quotes and double quotes characters.
![grafik](https://github.com/user-attachments/assets/de20c131-cbf2-469d-a9c7-3ab27270e93e)

Currently, the code is similar to:
https://github.com/FreshRSS/FreshRSS/blob/20c1d06e6258b211fb208fc885ad4e2d6e4a4094/app/layout/aside_feed.phtml#L129-L132
(the line breaks are a bit on different place, because there is an additional first line in `aside_feed.phtml` that is not there in subscription management page.)

While searching for a line break in `title` attribute Stackoverflow recommends mostly `'&#13;'`

I do not see a benefit of using `"\n"` instead of `'&#13;'`

Reply:
It does not matter much, so it can stay as you prefer. But what about `PHP_EOL` ?
I try to avoid entities and favour native characters whenever possible, which I find typically easier to read, lighter, and less error-prone.